src/App.js

```diff
@@ -11,12 +11,30 @@ import './styles/styles.scss';
 
 import DATA from './mockedData/MOCKED_DATA';
 
-const App = () => (
-  <ErrorBoundary>
-    <Header />
-    <Main requestedFilms={DATA} />
-    <Footer />
-  </ErrorBoundary>
-);
+class App extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            isDisplayFilmDescription: false,
+        };
+        this.changeState.bind(this);
+    }
+
+    changeState = () => {
+        let { isDisplayFilmDescription } = this.state;
+        this.setState({'isDisplayFilmDescription': !isDisplayFilmDescription})
+    };
+
+    render () {
+        let { isDisplayFilmDescription } = this.state;
+        return(
```

Comment:
seems your eslint don't have the rule which handle spaces and ';'. You also can add pre-push hook, which will ran eslint and test for you before push.